src/styles/utils/variable.scss

```diff
@@ -0,0 +1,27 @@
+// Nav
+$nav-hight: 60px;
+$nav-color: #00acdc;
+$nav-link-color: #060b35;
+
+// Card-grid
+$column-count: 1;
+$small-min-width: 488px;
+$medium-min-width: 768px;
+$large-min-width: 1024px;
+
+// Card-size
+$main-card-width: 100px;
+$percent-width-img: 0.6;
+$percent-hight-img: 0.34;
+$percent-wdith-content: 0.66;
```

Comment:
The variable `$percent-wdith-content` has a typo. It should be `$percent-width-content`.